pandunia-loge.csv

```diff
@@ -899,11 +899,11 @@ Eurasia|||eng:spa:may:Eurasia, por:Eurásia, fra:Eurasie, rus:Евразия (Ev
 eureka|||ell:ευρίσκω (evrísko), eng:spa:eureka, por:eureca, fra:eurêka, rus:эврика (evrika), tur:efreka, jpn:ユーレカ (yūreka)|find (discover)||econtrar (hallar)|||||||||||||trovi|löytää (keksiä)|znaleźć, znajdywać, odkryć, odkrywać
 euro||||euro (€)||euro|||||||||||||eŭro|euro (€)|euro (€)
 Europa|planete 5 lun 2|||Europa||||||||||||||||Europa-kuu|
+europa yum|yum 063|Eu|eng:europium, fra:europium, spa:europio, por:európio, rus:европий, jpn:ユウロピウム, kor:유로퓸, vie:europi, hin:युरोपियम, ben:ইউরোপিয়াম, may:europium, swa:europi|europium|europium|europio|európio|европий|يروبيوم|铕|ユウロピウム|유로퓸|europi|युरोपियम|ইউরোপিয়াম|europium|europi|europyum|eŭropio|europium|europ
```

Comment:
wouldn't **europe yum** make more sence, since it's named for the continent, not the moon?

Reply:
Good point. I thought that both are "Europa".